src/views/accounts/AccountPush/ExecutionDetail/AccountPushRecord.vue

```diff
@@ -80,14 +80,15 @@ export default {
       headerActions: {
         hasSearch: true,
         hasRefresh: true,
-        hasLeftActions: false,
+        hasLeftActions: true,
         hasRightActions: true,
         hasExport: false,
         hasImport: false,
         hasCreate: false,
         hasBulkDelete: false,
         hasBulkUpdate: false,
         searchConfig: {
+          getUrlQuery: true,
           exclude: ['id', 'status', 'execution'],
           options: [
             {
```

Comment:
The code appears to be missing an `export * from './';` directive at the end of each file, which is generally used when exporting modules that can import into other files. It would also help if you could provide more context about what specific parts (besides the difference between `hasLeftActions` and `false`) need clarification or optimization.